test/mongodb.test.js

```diff
@@ -1,9 +1,46 @@
 // This test written in mocha+should.js
 var semver = require('semver');
 var should = require('./init.js');
+var initialize = require('../lib/mongodb.js').initialize;
+var createDS = require('loopback-datasource-juggler').DataSource;
```

Comment:
you can remove `// This test written in mocha+should.js` too while you're at it ;)